### ISSUE TITLE
Is there a way to adjust the view distance?

### ISSUE THREAD

**Issue by Tasosgemah** [wipeout-phantom-edition/wipeout-phantom-edition#14]
Seems like the view distance is improved.... But why not go all the way and make it infinite so you can see the whole track? Something like this:

http://phoboslab.org/wipeout/

I'm sure any device released last 20 years can handle this with ease. At least a slider to control this would be nice.

**Comment by Matsilagi** [comment]
Tracks look ugly with you seeing all its missing faces from anywhere.

**Comment by Tasosgemah** [comment]
> Tracks look ugly with you seeing all its missing faces from anywhere.

I get what you are saying but is it really uglier than the track appearing in front of you?

Maybe there could be some fog in selected areas where this problem is more is more apparent? Because most of the time it looks pretty good.

**Comment by Defreezer** [comment]
I'd really like to have that as an option as well. I grew up with these games, don't really mind how it looks. Just want to see all of it.
Psygnosis devs actually designed the tracks in Wipeout in a way that hides the draw distance as much as possible, so that players wouldn't see the track pop-in from a distance while playing. Having a maximum draw distance option would've been 100% in their original vision.

**Comment by Baklojan** [comment]
This game actually would work really great with full draw distance, I don't mind missing faces because of how ps1 games look, so having that feature would be great

**Comment by LichKing1523** [comment]
Yeah, as an option, I think would be great.

**Comment by bonkmaykrQ** [comment]
Yeah. This was one of the first things I was disappointed the port didn't have at launch, it would be nice to see it implemented soon

**Comment by manic232** [comment]
Agreed 0 draw distance would be a great move for this superb port
